### PR TITLE
Japanese Translation Part I

### PR DESCRIPTION
**Pull request by totti**
* Done some ja translation.
* Language option for Japanese is hidden so that I can continue translation in background.
* Merged DT updates from main branch.
  * Merging caused some indentation mismatch between en/zh and ja. will fix them in next pr.

### CODE REVIEW

**Comment by totti** [comment]
@miyehn 
Could you please review this pr?

**Review by miyehn — COMMENTED** [review]
(I only just realized the comment I wrote yesterday was not submitted...)

File: src/Components/Skills.js
[the remark this answers is not in the document]
Comment (totti) [reply src/Components/Skills.js:1]: Sure. Will do 👍

[the remark this answers is not in the document]
Comment (totti) [reply src/Components/Skills.js:1]: Pushed new commit.
Hopefully, the diff now contains only changes related to translations.

**Pull request merged**